lib/shared/addon/components/node-selector/template.hbs

```diff
@@ -0,0 +1,81 @@
+<div class="mb-20"><label class="acc-label">{{t title}}</label></div>
+
+{{#if ruleArray.length}}
+  <table class="table fixed no-lines mb-20">
+    {{#if editing}}
+      <thead>
+        <tr>
+          <th class="acc-label" >{{t 'formScheduling.key'}}</th>
+          <th width="40"></th>
+          <th class="acc-label">{{t 'formScheduling.operator'}}</th>
+          <th width="40"></th>
+          <th class="acc-label">{{t 'formScheduling.value'}}</th>
+          <th width="10">&nbsp;</th>
+          <th width="40"></th>
+        </tr>
+      </thead>
+    {{/if}}
+    <tbody>
+      {{#each ruleArray as |rule|}}
+        <tr>
+          {{#if (gte rule.custom.length 0)}}
+            <td colspan="5">
+              {{#input-or-display editable=editing value=rule.custom}}
+                {{input type="text" class="form-control input-sm" placeholder=(t 'formScheduling.nodeSelector.custom.placeholder') value=rule.custom}}
+              {{/input-or-display}}
+            </td>
+          {{else}}
+            <td>{{input type="text" class="form-control input-sm" value=rule.key}}</td>
+            <td>&nbsp;</td>
+            <td>
+              {{new-select
+                class="form-control input-sm"
+                content=operatorChoices
+                localizedLabel=true
+                value=rule.operator
+                readOnly=editing
+                style="display: inline-block; width: auto;"
+              }}
+            </td>
+            <td>&nbsp;</td>
+            <td>
+              {{#if (and (not-eq rule.operator 'Exists') (not-eq rule.operator 'DoesNotExist'))}}
```

Comment:
Add computed properties for these and flip the order, they don't need to be nested conditions..

```
{{#if isJustLabel}}
  generic.na
{{else if isMultiple}}
  input with multiple placeholder
{{else}}
  input with no placeholder
{{/if}}
```

```
  isJustLabel: computed('rule.operator', function() {
    return ['Exists','DoesNotExist'].includes(get(this, 'rule.operator'));
  }),
  isMultiple: computed('rule.operator', function() {
    return ['In','NotIn'].includes(get(this, 'rule.operator'));
  }),
```